modules/compiler/utils/include/compiler/utils/metadata.h

```diff
@@ -269,6 +269,11 @@ struct KernelInfo {
 void populateKernelList(llvm::Module &m,
                         llvm::SmallVectorImpl<KernelInfo> &results);
 
+/// @brief Replaces instances of kernel fromF with toF in module-level
+/// !opencl.kernels metadata.
```

Comment:
should probably have @param too, although I'm not sure how much that gives you here.

Reply:
aye right you are. doesn't hurt though and it's probably not for me to decide whether something's too obvious to document?